config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -567,7 +567,7 @@ presubmits:
       preset-kind-volume-mounts: "true"
       preset-azure-community: "true"
     decorate: true
-    run_if_changed: '^test\/e2e\/(config\/azure-dev\.yaml)|(data\/shared\/v1beta1_provider\/metadata\.yaml)$'
+    run_if_changed: '^(test\/e2e\/(config\/azure-dev\.yaml|data\/shared\/v1beta1_provider\/metadata\.yaml)|azure\/scope\/.*)$'
```

Comment:
I'm concerned about the `^` and `$` here now that we're adding the open-ended directory match to the end (I don't think we need to be that strict)

`test\/e2e\/(config\/azure-dev\.yaml)|(data\/shared\/v1beta1_provider\/metadata\.yaml)|azure\/scope` is probably sufficient

Reply:
I tested the regex here https://regex101.com/r/ah1HgX/2
`^` and `$` are necessary to ensure we only match `test/e2e/config/azure-dev.yaml` **or** `test/e2e/data/shared/v1beta1_provider/metadata.yaml`. Any changes in `azure/scope/` still work.

Reply:
regex101.com si better than my personal concern :)